README.md

```diff
@@ -61,6 +61,10 @@ ros2 launch continental_ars408_socket_can.launch.xml
   - The bool parameter to determine output publishing behavior.
   - If this parameter is set to false (default value), the driver will publish output after receiving a complete cycle of sequential data from the CAN data topic.
   - If this parameter is set to true, the driver will publish output every time data is received from the CAN data topic.
+- `size_x`
+  - The size of x [m]
+- `size_y`
+  - The size of y [m]
```

Comment:
Could you add some explanation that the default size is 3.5/1.5 [m]

Reply:
It means a value close to car size.
Considering use of [multi_objects_tracker](https://github.com/autowarefoundation/autoware.universe/tree/main/perception/multi_object_tracker), `min_area_matrix` parameters of it needs to be considered.
This PR set to this value to exceed default value of `min_area_matrix` of multi_objects_tracker.

The background of default value for now is above, but I think again for default value and it should be defined by resolution of ARS408 radar.
Because distance resolution measuring of ARS408 is 1.79 m for far range, I think the default size should be set 1.8m / 1.8m.
(For practical use for projects, to use multi_objects_tracker and make easy to visualize yaw angle of objects, the value like 3.5 / 1.5 [m] may be useful.)
What do you think of this opinion?

Reply:
As we have discussed on a mtg, it feels weird to consider the parameters of tracker at driver here. So, 

> default size should be set 1.8m / 1.8m.

It sounds good to me.

Reply:
fixed at [661693d](https://github.com/tier4/ars408_driver/pull/32/commits/661693d7e5df77e7fb4051a4fdef7e48845ceca4)